install_config/topics/using_gcp_load_balancer.adoc

```diff
@@ -0,0 +1,119 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+[id='gce-load-balancer_{context}']
+= Using the GCP external load balancer as a service
+
+{product-title} can leverage the GCP load balancer capacity by exposing services
+externally using a `LoadBalancer` service. {product-title} creates the load
+balancer in GCP and creates the proper firewall rules.
+
+The following is an example `LoadBalancer` service:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: hello-openshift
+  name: hello-openshift-external
+spec:
+  externalTrafficPolicy: Cluster
+  ports:
+  - name: port-1
+    nodePort: 30714
+    port: 8080
+    protocol: TCP
+    targetPort: 8080
+  - name: port-2
+    nodePort: 30122
+    port: 8888
+    protocol: TCP
+    targetPort: 8888
+  selector:
+    app: hello-openshift
+    deploymentconfig: hello-openshift
+  sessionAffinity: None
+  type: LoadBalancer
+----
+
+.Procedure
+
+. Create a new application:
++
+[source,bash]
+----
+$ oc new-app openshift/hello-openshift
+----
+
+. Expose the load balancer service:
++
+[source,bash]
+----
+$ oc expose dc hello-openshift --name='hello-openshift-external' --type='LoadBalancer'
+----
+
+. To verify that the service has been created:
++
+[source,bash]
+----
+$ oc get svc
+NAME                       TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)                         AGE
+hello-openshift            ClusterIP      172.30.62.10     <none>          8080/TCP,8888/TCP               20m
+hello-openshift-external   LoadBalancer   172.30.147.214   35.230.97.224   8080:31521/TCP,8888:30843/TCP   19m
+----
++
+The `LoadBalancer` type and External IP fields indicate that the service is
+using GCP load balancers to expose the application.
+
+{product-title} creates the required objects in the GCP infrastructure such as:
```

Comment:
Are we confirming that these have been created? Then they should be steps. Are we just talking about the cool stuff that OCP does? Then it should go in a concept.

Reply:
This is to confirm that everything required has been created in GCP